wire-ios-sync-engine/Source/Synchronization/Strategies/UserClientRequestStrategy.swift

```diff
@@ -396,7 +403,10 @@ public final class UserClientRequestStrategy: ZMObjectSyncStrategy, ZMObjectStra
         switch response.result {
         case .success:
             if let payload = response.payload?.asArray() as? [[String: AnyObject]] {
-                self.received(clients: payload)
+                // FIXME: CC check if ok async
```

Comment:
It would not be OK if there are any request completion handlers attached to the request.